valuable/src/named_values.rs

```diff
@@ -22,6 +22,16 @@ impl<'a> NamedValues<'a> {
         self.values.get(idx)
     }
 
+    pub fn get_by_name(&self, name: &str) -> Option<&Value<'_>> {
```

Comment:
nit/TIOLI: should: `name` be
```suggestion
    pub fn get_by_name(&self, name: impl AsRef<str>) -> Option<&Value<'_>> {
```
or something? It could even be
```suggestion
    pub fn get_by_name<Q>(&self, name: Q) -> Option<&Value<'_>> 
    where
        Q: PartialEq<str>
    {
```